### DISCUSSION TITLE
Disabling virtual sites

### DISCUSSION THREAD

**Discussion by freakyacorn** [marrink-lab/polyply_1.0#376] [Q&A]
Is it possible to disable the production of virtual sites when generating a .itp file for a protein sequence?

**Comment by fgrunewald** [comment]
Technically speaking you can use an [old ff-file](https://github.com/marrink-lab/polyply_1.0/blob/5d97c832b80250196a8c175dde56da4ad3a4b566/polyply/data/martini3/aminoacids.ff). However, the VS is an integral part of the new model as it facilitates the water scaling. If you decide to change the water BB interactions instead be aware that this might effect other molecules in your system. Thus we strongly recommend to not do this.

**Reply by ywitzky** [reply]
@fgrunewald  My code emits "KeyError: frozenset({'VS'})" since i have upgraded polyply, which i think is due to the particle type VS not being defined. Including some random parameters like:
`[ atomtypes ]\n
 VS 72.0 0.000 A 0.0 0.0`

into the *.top files fixes the error message. Can you provide reasonable parameters for the VS type which is used in aminoacids.ff for martini3 or tell me the missing files that i forgot to include? I do not know the Martini model well enough, since i only use it to generate starting configurations of dense protein systems for united residue models.

This might have been the reason why freakyacorn wanted to remove them.

**Comment by fgrunewald** [comment]
@csbrasnett do you have an idea what is happening here?

**Reply by csbrasnett** [reply]
@fgrunewald @ywitzky my best guess is that it's not enough to include the VS in the `[atomtypes]` directive, there also need to be an entry in `martini_v3.0.0.itp` to indicate the nonbonded interactions. Alternatively with the latest main branch of polyply, you can use the martini3IDP force field, which doesn't have any virtual site stuff to worry about for building a system

**Reply by csbrasnett** [reply]
@ywitzky apologies, I just reread your reply! Yes, the atomtype definitely needs to be included for this model ( the martini3-go force field) - please see the [Go tutorial](https://vermouth-martinize.readthedocs.io/en/latest/tutorials/go_models.html) for more details about the practical use of the Gō model. For the purposes of this model, you can follow the tutorial [here](https://github.com/marrink-lab/polyply_1.0/wiki/Tutorial:-Martini-3-IDPs---proteins#using-idp-topologies) for appropriate parameters.
